Source/WebCore/platform/audio/gstreamer/AudioDecoderGStreamer.cpp

```diff
@@ -261,6 +262,7 @@ GStreamerInternalAudioDecoder::GStreamerInternalAudioDecoder(const String& codec
         harnessedElement = WTFMove(element);
 
     m_harness = GStreamerElementHarness::create(WTFMove(harnessedElement), [weakThis = WeakPtr { *this }, this](auto&, GRefPtr<GstSample>&& outputSample) {
+        RefPtr protectedThis = weakThis.get();
         if (!weakThis)
```

Comment:
Should we check `protectedThis` here now?

Reply:
Ah right, that would be better!

Reply:
Done